internal/compiler/widgets/cosmic/tabwidget.slint

```diff
@@ -140,10 +140,12 @@ export component TabBarImpl inherits TabBarBase {
     accessible-delegate-focus: root.current-focused >= 0 ? root.current-focused : root.current;
     preferred-height: 44px;
 
-    HorizontalLayout {
-        @children
+    Flickable {
+        HorizontalLayout {
+            @children
+        }
```

Comment:
I think that this should just be a clip, not a full fledged `Flickable` with actual behaviour. (here and in the other files).

```suggestion
    Rectangle {
        clip: true;
        HorizontalLayout {
            @children
        }
```

Reply:
Ah yes. It feels a bit weird to pan, but it's indeed much better than not accessible tabs at all :)